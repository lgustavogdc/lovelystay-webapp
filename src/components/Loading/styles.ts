import styled from 'styled-components'

export const LoadingWrapper = styled.div.attrs({
  'data-testid': 'LoadingWrapper',
})``
