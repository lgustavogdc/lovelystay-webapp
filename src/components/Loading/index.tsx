import * as S from './styles'
import CircularProgress from '@mui/material/CircularProgress'

const Loading = () => (
  <S.LoadingWrapper>
    <CircularProgress />
  </S.LoadingWrapper>
)

export default Loading
