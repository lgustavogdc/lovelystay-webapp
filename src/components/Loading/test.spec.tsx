import { render, screen } from '@testing-library/react'

import Loading from '.'

describe('<Loading />', () => {
  it('should render the loading', () => {
    render(<Loading />)
    const loading = screen.getByTestId('LoadingWrapper')

    expect(loading).toBeInTheDocument()
  })
})
