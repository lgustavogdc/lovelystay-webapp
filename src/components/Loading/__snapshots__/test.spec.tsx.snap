// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render the loading 1`] = `
<div
  class="sc-bdvvtL"
>
  <h1>
    Loading
  </h1>
</div>
`;
